[new addon] zigbee-conbee-adapter

Adapter using ConBee USB stick to access Zigbee devices.

```
{
    "name": "zigbee-conbee-adapter",
    "display_name": "Zigbee adapter ConBee USB stick",
    "description": "Adapter for Zigbee  using  ConBee USB stick",
    "version": "0.9.0",
    "url": "https://github.com/tomasy/zigbee-conbee-adapter/releases/download/0.9.0/zigbee-conbee-adapter-0.9.0.tgz",
     "api": {
      "min": 2,
      "max": 2
    }
}
```

PS.
Is it possible to change description for my DateTime Adapter
from _A DateTime adapter to create more advanced rules_ to
_A DateTime adapter to create more advanced rules, Supports e.g. sunrise, sunset, weekends,..._

Fixed your comments, but I noticed that after a `send_error` and exit(100) the adapter is still enabled in the gateway. 
And a more serious problem. If I enable it I sometimes get a ` Segmentation fault` and the gateway restarted.
https://github.com/tomasy/zigbee-conbee-adapter/releases/download/0.9.1/zigbee-conbee-adapter-0.9.1.tgz

```
nov 04 11:42:02 gateway run-app.sh[4348]: 2018-11-04 11:42:02.504 zigbee-conbee: conbee_adapter.py:68 ERROR Problem during pairing of devices. Will exit. Execption: unknown url type: 's/api/E09EC9XXXX//lights'                    
nov 04 11:42:02 gateway run-app.sh[4348]: 2018-11-04 11:42:02.964 zigbee-conbee: AddonManagerProxy: Failed to send message: Bad file descriptor                                                                                      
nov 04 11:42:05 gateway run-app.sh[4348]: 2018-11-04 11:42:05.450 zigbee-conbee: conbee_adapter.py:89 INFO End unload all devices                           
nov 04 11:42:05 gateway run-app.sh[4348]: 2018-11-04 11:42:05.452 zigbee-conbee: AddonManagerProxy: Failed to send message: Nanomsg library was terminated          
nov 04 11:42:13 gateway run-app.sh[4348]: 2018-11-04 11:42:13.113 Plugin: zigbee-conbee-adapter died, code = 100 NOT restarting...                                                                                                                          
nov 04 11:42:31 gateway run-app.sh[4348]: 2018-11-04 11:42:31.192 Unloading ConBeeAdapter                                                                                                                                                           
nov 04 11:42:31 gateway run-app.sh[4348]: Segmentation fault                                                                                                       
nov 04 11:42:31 gateway run-app.sh[4348]: npm ERR! code ELIFECYCLE                                                                                      
nov 04 11:42:31 gateway run-app.sh[4348]: npm ERR! errno 139                                                                                                                                                              
nov 04 11:42:31 gateway run-app.sh[4348]: npm ERR! things-gateway@0.6.1 start: `webpack && node build/gateway.js "--check-wifi"`                                   
nov 04 11:42:31 gateway run-app.sh[4348]: npm ERR! Exit status 139                                                                                                             
nov 04 11:42:31 gateway run-app.sh[4348]: npm ERR!                                                                                                         
nov 04 11:42:31 gateway run-app.sh[4348]: npm ERR! Failed at the things-gateway@0.6.1 start script.                                                                                                                                                            
nov 04 11:42:31 gateway run-app.sh[4348]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.            
nov 04 11:42:31 gateway run-app.sh[4348]: npm ERR! A complete log of this run can be found in:                                                                  
nov 04 11:42:31 gateway run-app.sh[4348]: npm ERR!     /home/pi/.npm/_logs/2018-11-04T10_42_31_282Z-debug.log                                                                                                              
nov 04 11:42:41 gateway systemd[1]: mozilla-iot-gateway.service: Service hold-off time over, scheduling restart.                                                                      
nov 04 11:42:41 gateway systemd[1]: Stopped Mozilla IoT Gateway Client.                                                                                                                                                                  
nov 04 11:42:41 gateway systemd[1]: Started Mozilla IoT Gateway Client.                                                                                                                                                        



/home/pi/.npm/_logs/2018-11-04T10_42_31_282Z-debug.log

0 info it worked if it ends with ok
1 verbose cli [ '/home/pi/.nvm/versions/node/v8.12.0/bin/node',
1 verbose cli   '/home/pi/.nvm/versions/node/v8.12.0/bin/npm',
1 verbose cli   'start',
1 verbose cli   '--',
1 verbose cli   '--check-wifi' ]
2 info using npm@6.4.1
3 info using node@v8.12.0
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle things-gateway@0.6.1~prestart: things-gateway@0.6.1
6 info lifecycle things-gateway@0.6.1~start: things-gateway@0.6.1
7 verbose lifecycle things-gateway@0.6.1~start: unsafe-perm in lifecycle true
8 verbose lifecycle things-gateway@0.6.1~start: PATH: /home/pi/.nvm/versions/node/v8.12.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/pi/mozilla-iot/gateway/node_modules/.bin:/home/pi/.nvm/versions/node/v8.12.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/u\
sr/bin:/sbin:/bin
9 verbose lifecycle things-gateway@0.6.1~start: CWD: /home/pi/mozilla-iot/gateway
10 silly lifecycle things-gateway@0.6.1~start: Args: [ '-c', 'webpack && node build/gateway.js "--check-wifi"' ]
11 silly lifecycle things-gateway@0.6.1~start: Returned: code: 139  signal: null
12 info lifecycle things-gateway@0.6.1~start: Failed to exec start script
13 verbose stack Error: things-gateway@0.6.1 start: `webpack && node build/gateway.js "--check-wifi"`
13 verbose stack Exit status 139
13 verbose stack     at EventEmitter.<anonymous> (/home/pi/.nvm/versions/node/v8.12.0/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at EventEmitter.emit (events.js:214:7)
13 verbose stack     at ChildProcess.<anonymous> (/home/pi/.nvm/versions/node/v8.12.0/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at ChildProcess.emit (events.js:214:7)
13 verbose stack     at maybeClose (internal/child_process.js:915:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
14 verbose pkgid things-gateway@0.6.1
15 verbose cwd /home/pi/mozilla-iot/gateway
16 verbose Linux 4.14.30-v7+
17 verbose argv "/home/pi/.nvm/versions/node/v8.12.0/bin/node" "/home/pi/.nvm/versions/node/v8.12.0/bin/npm" "start" "--" "--check-wifi"
18 verbose node v8.12.0
19 verbose npm  v6.4.1
20 error code ELIFECYCLE
21 error errno 139
22 error things-gateway@0.6.1 start: `webpack && node build/gateway.js "--check-wifi"`
22 error Exit status 139
23 error Failed at the things-gateway@0.6.1 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 139, true ]


```

--- a/list.json
+++ b/list.json
@@ -903,6 +903,33 @@
     ]
   },
   {
+    "name": "zigbee-conbee-adapter",
+    "display_name": "Zigbee (ConBee USB stick)",
+    "description": "Uses deCONZ REST API to manage ConBee USB stick",
+    "author": "tomasy",
+    "homepage": "https://github.com/tomasy/zigbee-conbee-adapter",
+    "packages": [
+      {
+        "architecture": "any",
+        "language": {
+          "name": "python",
+          "versions": [
+            "3.5",
+            "3.6",
+            "3.7"
+          ]
+        },
+        "version": "0.9.5",
+        "url": "https://s3-us-west-2.amazonaws.com/mozilla-gateway-addons/zigbee-conbee-adapter-0.9.5.tgz",
+        "checksum": "d060a7faf508ab8f553a2e100e48cfcdf5688d483daacf1e4e6b554b9d9028ca",
+        "api": {
+          "min": 2,
+          "max": 2
+        }
+      }
+    ]
+  },
+  {
     "name": "zwave-adapter",
     "display_name": "Z-Wave",
     "description": "Z-Wave device support",

